systemtest/src/test/java/io/strimzi/systemtest/operators/topic/ThrottlingQuotaST.java

```diff
@@ -0,0 +1,389 @@
+            LOGGER.info("Executing {}/{} iteration for {}.", i + 1, iterations, deleteAdminName);
+            deleteAdminClientJob = deleteAdminClientJob.toBuilder()
+                    .withTopicOffset(i * offset)
+                    .withTopicCount(100)
+                    .withAdditionalConfig("")
+                    .build();
+            resourceManager.createResource(extensionContext, deleteAdminClientJob.defaultAdmin().build());
+            ClientUtils.waitForClientSuccess(deleteAdminName, NAMESPACE, 10);
+            JobUtils.deleteJobWithWait(NAMESPACE, deleteAdminName);
+        }
+    }
+
+    @ParallelTest
+    void testThrottlingQuotasCreateAlterPartitions(ExtensionContext extensionContext) {
+        final String kafkaUsername = kafkaUsernamePrefix + "-" + new Random().nextInt(Integer.MAX_VALUE);
+        final String createAdminName = "create-admin-client" + "-" + new Random().nextInt(Integer.MAX_VALUE);
+        final String alterAdminName = "alter-admin-client" + "-" + new Random().nextInt(Integer.MAX_VALUE);
+        final String topicNamePrefix = classTopicPrefix + "-partitions";
+        int topicsCount = 50;
+        int topicPartitions = 100;
+        setupKafkaUserInNamespace(extensionContext, kafkaUsername);
+
+        KafkaAdminClient adminClientJob = new KafkaAdminClient.Builder()
+                .withAdminName(createAdminName)
+                .withBootstrapAddress(KafkaResources.plainBootstrapAddress(CLUSTER_NAME))
+                .withTopicName(topicNamePrefix)
+                .withTopicCount(topicsCount)
+                .withPartitions(topicPartitions)
+                .withNamespaceName(NAMESPACE)
+                .withTopicOperation(AdminClientOperations.CREATE_TOPICS.toString())
+                .withAdditionalConfig(getAdminClientConfig(kafkaUsername))
+                .build();
+        resourceManager.createResource(extensionContext, true, adminClientJob.defaultAdmin().build());
+        String createPodName = kubeClient().listPods("job-name", createAdminName).get(0).getMetadata().getName();
+        PodUtils.waitUntilMessageIsInPodLogs(
+                createPodName,
+                "org.apache.kafka.common.errors.ThrottlingQuotaExceededException: The throttling quota has been exceeded.",
+                Duration.ofMinutes(5).toMillis()
+        );
+        JobUtils.deleteJobWithWait(NAMESPACE, createAdminName);
+
+        // Delete created topics (as they were created in random order, we have to use KafkaTopic instead of this client)
+        List<KafkaTopic> ktl = KafkaTopicUtils.getAllKafkaTopicsWithPrefix(NAMESPACE, topicNamePrefix);
+        for (KafkaTopic kt : ktl) {
+            cmdKubeClient().deleteByName(KafkaTopic.RESOURCE_SINGULAR, kt.getMetadata().getName());
+        }
+
+        // Throttling quota after performed 'alter' partitions on existing topic
+        int topicAlter = 20;
+        adminClientJob = new KafkaAdminClient.Builder()
```

Comment:
How about `toBuilder()` again? Because you are still loading there same parameters